jobs/ci-kubernetes-e2e-gce-canary.env

```diff
@@ -0,0 +1,4 @@
+# Everything else is the same as the other job it is testing
+BOOTSTRAP_METADATA_CANARY_IMAGE=true
```

Comment:
I cannot find this `BOOTSTRAP_METADATA_CANARY_IMAGE` anywhere else? It's just some env you want to define?